Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Types/SearchRequestForDriver.hs

```diff
@@ -92,8 +92,8 @@ data SearchRequestForDriverAPIEntity = SearchRequestForDriverAPIEntity
     durationToPickup :: Seconds,
     baseFare :: Money,
     customerExtraFee :: Maybe Money,
-    fromLocation :: DLoc.SearchReqLocation,
-    toLocation :: DLoc.SearchReqLocation,
+    fromLocation :: DLoc.Location,
+    toLocation :: DLoc.Location, -- we need to show all requests or last one ?
```

Comment:
It wouldn't backward compatible if we will show all locations. Maybe add separate field for all `toLocations`? @hkmangla

Reply:
Yes @roman-bodavskiy we need to discuss these changes for multiple locations . I have just commented out where I was having doubt since I am expecting to do these changes at a later stage when I will work on adding multiple stops after completing edit pickup and edit destination.

@roman-bodavskiy @hemantmangla please suggest how can we handle them.